docs/metrics/coefficient-of-determination.md

```diff
@@ -0,0 +1,77 @@
+# Coefficient of Determination (R²)
+
+The Coefficient of Determination, commonly denoted as R², is a statistical measure used to assess the goodness of fit
+of a regression model. It represents the proportion of the variance in the dependent variable that is predictable from
+the independent variables.
+
+R² provides a scale that is **generally** from 0 to 1, where higher values indicate a better fit and imply that
+the model can better explain the variation of the output with the input variables. It is particularly useful for
```

Comment:
suggest using `ground truth` and `inference` instead.

Reply:
reworded (also discussed in previous thread)